lib/packets/resultset_header.js

```diff
@@ -18,6 +18,15 @@ function ResultSetHeader (packet, bigNumberStrings, encoding)
     this.infileName = packet.readString(undefined, encoding);
   }
 
+  // issue#288: Add changedRow if exists, reference to https://github.com/sidorares/node-mysql2/issues/288
+  if (encoding) {
+    var message = packet.readString(undefined, encoding);
+    var m = message.match(/\schanged:\s*(\d+)/i);
```

Comment:
I'd argue that `changedRows` should not be part of api ( up to client how to parse message) but since it's already in node-mysql I'm OK to have it as well